ThermofluidStream/Processes/FlowResistance.mo

```diff
@@ -72,34 +74,28 @@ protected
     "Density of medium entering";
   SI.DynamicViscosity mu_in = Medium.dynamicViscosity(inlet.state)
     "Dynamic viscosity of medium entering";
-  SI.Length perimeterActual "Actual perimeter of resistance";
-  SI.Area areaCrossActual "Actual area of resistance";
-  SI.Length D_h "Hydraulic diameter of resistance";
+  parameter SI.Length D_h = 4*areaCrossActual/perimeterActual "Hydraulic diameter of resistance";
+
+  parameter SI.Length perimeterActual=
+    if shape == ShapeOfResistance.circular then 2*pi*r
+    elseif shape == ShapeOfResistance.rectangle then 2*a+2*b
+    else perimeter "Actual perimeter of resistance";
+
+  parameter SI.Area areaCrossActual=
+    if shape == ShapeOfResistance.circular then pi*r*r
+    elseif shape == ShapeOfResistance.rectangle then a*b
+    else areaCross "Actual area of resistance";
 
   parameter SI.Area areaInertance=
-    if shape == ThermofluidStream.Processes.Internal.ShapeOfResistance.circular then pi*r*r
-    elseif shape == ThermofluidStream.Processes.Internal.ShapeOfResistance.rectangle then (2*a*b)/(a+b)
+    if shape == ShapeOfResistance.circular then pi*r*r
+    elseif shape == ShapeOfResistance.rectangle then pi*D_h*D_h*1/4
```

Comment:
`pi*D_h*D_h*1/4` seems to apply also for shape == circular. So either retain the original equations (`pi*r*r` for circular and `(2*a*b)/(a+b)` for rectangular) or simplify accordingly.